README.md

```diff
@@ -223,4 +232,77 @@ some specific game's data (with some pitchers and batters omitted for brevity)
     }
   }
 ]
-```
\ No newline at end of file
+```
+
+## Prediction Model
+
+The prediction model employs the notion of Deep Learning to predict the result from the data. 
+
+The scraped data is parsed in the formatter.py to follow the below format:
+```
+hometeam's data = The summary + standing of the team's previous k games # k is the usr input.
+awayteam's data = The summary + standing of the team's previous k games
+
+result of the game = [x, y] 
+``` 
+
+The builder.py constructs the neural network that takes in the above format of data to be trained and predict the result of the game.
+
+When taking the individual team data, as we are not distinguishing the team specific statistics, auto encoder & decoder were used to get rid of such uncertainty then passed on to neural network.
+
+The number of layers can't be decided by the user yet but we are planning on letting the user take care of such details.
+
+### Note
+- Each layer of neural network will perform dropout according to the usr input.  
+- SeLU and AdamOptimzer were used in the architecture.  
+
+### Getting Started
+To run the code you are required to use python3. I suggest following the below instruction to run the code.
+```
+virtualenv -p python3 .env
+source .env/bin/activate       
+pip install -r requirements.txt  
+# Run the code
+deactivate                       # Exit the virtual environment
+```
+
+### Predicting the model
+After activating the virtual envelop, run  
+```
+python trainer.py -h
+
+# Result
+KBO Score Prediction Trainer
+
+positional arguments:
+  year             The year of data to train the model with
+  train_size       The proportion of the training set to the test set
+  model_name       The name of the model
+  learn_rate       The learning rate
+  sequence_length  Sequence length
+  epoch            Training epoch
+  drop_rate        Drop rate
+
+optional arguments:
+  -h, --help       show this help message and exit
+```
+to see what the user needs to provide.  
+
+The model takes in 7 inputs from the user.
+1. The year of the game to train the Deep Learning model with.
+2. The proportion of the training set to the test set in the raw dataset.
+3. The name of the model. (For saving purpose)
+4. The learning rate of the model.
+5. The number of previous games the model should look at for each game.
+6. The iteration of training.
+7. The drop rate.
+
+### Saving the trained model
```

Comment:
It will be saved in saved_graph directory. The format of the file is cpkt (checkpoint in short) that has meta data for the trained tensor graph. When you want to load the trained model, you construct the same graph and simply call the trained meta data. This can be done easily since the builder will automatically construct the same graph for you and then call the meta data when it's told to predict. 1) 2) 3) will result in less data I'm afraid and that's why I wanted the model to not distinguish teams when trained. The auto encoder and decoder were used to handle that.